frontend/pages/hosts/details/cards/Queries/_styles.scss

```diff
@@ -33,9 +33,16 @@
               width: 0;
             }
           }
+
```

Comment:
@RachelElysia [the original spec](https://www.figma.com/design/MoP51amR79dMPoNnARjyTw/%2314415-See-latest-query-results-per-host?m=dev&node-id=230%3A13882&t=iNsZm766G6YKiW6l-1) for this feature maintains the arrows even when not hovering if a query has (or will once its collected) link to a query report. So item 2 isn't a bug, but I see your point – sounds like a product conversation to have with @rachaelshaw.

Reply:
Yep, the arrows are there to help users see which rows have more information, since this is a spot in the UI where some rows are clickable and some are not (which isn't really something we have elsewhere, hence the slightly different UI pattern).

No hover state for non-clickable rows was in the original spec though, so I'm glad you caught that 👍 
<img width="999" alt="Screenshot 2024-05-21 at 1 39 30 PM" src="https://github.com/fleetdm/fleet/assets/3065949/23eaa15f-b2c9-4784-850d-b65b8c9e6470">

Reply:
thanks @jacobshandling, fixing now.

Reply:
Sweet, updated the PR to only address the first bug as the second one is actually intended.

@jacobshandling @rachaelshaw qq: how can I get a query to show, but not show an arrow? I tried discarding data for a specific query but then it removed the query from the table all together.

Reply:
@RachelElysia I think if you enable automations for a query that's discarding data, or if you make a new query and it hasn't had a chance to run on the host yet. (Possibly also if you include a query in an old style pack, though I'm not 100% sure on that one.)

Reply:
@rachaelshaw I tried a new query that hasn't run on the host but I got a link to an empty state:

<img width="1388" alt="Screenshot 2024-05-21 at 4 16 33 PM" src="https://github.com/fleetdm/fleet/assets/71795832/2588e393-8340-4eef-ac39-a824e23e72b7">

Reply:
@RachelElysia https://www.figma.com/design/MoP51amR79dMPoNnARjyTw/%2314415-See-latest-query-results-per-host?m=dev&node-id=230%3A13890&t=iNsZm766G6YKiW6l-1

Reply:
OK after reviewing the designs, I think it's when a query has discard data enabled **and** automations enabled that it's visible but not clickable.

Looking at the original wireframes, I'm seeing 3 different empty states for "Report updated":
+ "No report" -> Query has discard data enabled and automations enabled (row not clickable)
+ "_Never_" -> Query hasn't run yet (row clickable)
+ "---" -> ??? (row not clickable)

(...judging by the older iterations on the Scratchpad page, I think we replaced "---" with "No report" and I forgot to update it in the other rows of the final design. I'll fix that now in case we ever need to refer back to these.)